<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1291</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color:#00555F;
	color: #000000;

}

</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>1281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>gestion employee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: #00555F ;
   border: none;
   color: #fff;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px; 
}

QPushButton:hover {
   background-color: #EF829B;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>291</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
   background-color: #f2f2f2;
   font-size: 14px;
   padding: 20px;
   border-radius: 10px;
}

QFrame QListWidget {
   background-color: transparent;
   color: #333;
   font-size: 14px;
   padding: 20px;
}

QFrame QListWidget::item {
   padding: 10px;
   border-bottom: 1px solid #ddd;
}

QFrame QListWidget::item:selected {
   background-color: #007aff;
   color: #fff;
}

QFrame QListWidget::item:hover {
   background-color: #ddd;
}

QFrame QLabel {
   font-size: 18px;
   font-weight: bold;
   color: #333;
   margin-bottom: 20px;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color:  #EF829B;
}

</string>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color:   #EF829B;
}

</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color: #ddd;
}

QMainWindow {
   background-color: #f2f2f2;
}

QMenuBar {
   background-color: #333;
   color: #fff;
}

QMenuBar::item {
   background-color: #333;
   color: #fff;
}

QMenu {
   background-color: #333;
   color: #fff;
}

QMessageBox {
   font: bold 14px;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #EF829B;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color: #ddd;
}

QMainWindow {
   background-color: #f2f2f2;
}

QMenuBar {
   background-color: #333;
   color: #fff;
}

QMenuBar::item {
   background-color: #333;
   color: #fff;
}

QMenu {
   background-color: #333;
   color: #fff;
}

QMessageBox {
   font: bold 14px;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tab">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>130</y>
        <width>861</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_supp">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>520</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>520</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>l'id de lemplyee a suprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bouton_supp">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>520</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
   background-color: #f2f2f2;
   font-size: 14px;
   padding: 20px;
   border-radius: 10px;
}

QFrame QListWidget {
   background-color: transparent;
   color: #333;
   font-size: 14px;
   padding: 20px;
}

QFrame QListWidget::item {
   padding: 10px;
   border-bottom: 1px solid #ddd;
}

QFrame QListWidget::item:selected {
   background-color: #007aff;
   color: #fff;
}

QFrame QListWidget::item:hover {
   background-color: #ddd;
}

QFrame QLabel {
   font-size: 18px;
   font-weight: bold;
   color: #333;
   margin-bottom: 20px;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color:  #EF829B;
}

</string>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color:   #EF829B;
}

</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #333;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color: #ddd;
}

QMainWindow {
   background-color: #f2f2f2;
}

QMenuBar {
   background-color: #333;
   color: #fff;
}

QMenuBar::item {
   background-color: #333;
   color: #fff;
}

QMenu {
   background-color: #333;
   color: #fff;
}

QMessageBox {
   font: bold 14px;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="styleSheet">
          <string notr="true">QLabel {
   font: bold 14px;
   color: #EF829B;
}

QPushButton {
   background-color: #ccc;
   border: none;
   color: #333;
   padding: 10px 20px;
   text-align: center;
   text-decoration: none;
   display: inline-block;
   font-size: 14px;
   margin: 4px 2px;
   cursor: pointer;
   border-radius: 16px;
}

QPushButton:hover {
   background-color: #ddd;
}

QMainWindow {
   background-color: #f2f2f2;
}

QMenuBar {
   background-color: #333;
   color: #fff;
}

QMenuBar::item {
   background-color: #333;
   color: #fff;
}

QMenu {
   background-color: #333;
   color: #fff;
}

QMessageBox {
   font: bold 14px;
}</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>70</y>
        <width>641</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="le_add">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>add</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>email</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>fonction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>statut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>droit_daccses</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_satut">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_droit_daccess">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mdp">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>280</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>280</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>mot de passe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num_tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_fonction">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>frame_4</zorder>
     <zorder>frame_3</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>931</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="le_modifier">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>email</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>fonction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>statut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>droit_daccses</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_satut_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_droit_daccess_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mdp_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>280</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>280</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>mot de passe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num_tel_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num_tel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_fonction_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>671</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
